#include "optiminf/util/parameter_naming_util.h"

namespace optiminf
{
const std::string ParameterNamingUtil::ACCEPTABLE_CONSTRAINT_VIOLATION_ON_SOLVER_FAILURE_PARAM_NAME = "solver/acceptable_constraint_violation";
const std::string ParameterNamingUtil::ACCEPTABLE_SOLVER_FAILURES_PARAM_NAME = "solver/acceptable_solver_failures";

const std::string ParameterNamingUtil::CHECK_FOR_NON_FINITE_VALUES_PARAM_NAME = "debug/check_for_non_finite_values";

const std::string ParameterNamingUtil::COST_DEBUG_LEVEL_PARAM_NAME = "debug/cost_debug_level";
const std::string ParameterNamingUtil::CONSTRAINT_DEBUG_LEVEL_PARAM_NAME = "debug/constraint_debug_level";
}  // namespace optiminf
